sql/core/src/test/resources/sql-tests/inputs/like-all.sql

```diff
@@ -0,0 +1,42 @@
+CREATE OR REPLACE TEMPORARY VIEW like_all_table AS SELECT * FROM (VALUES
+  ('google', '%oo%'),
+  ('facebook', '%oo%'),
+  ('linkedin', '%in'))
+  as t1(company, pat);
+
+select company from like_all_table where company like all ('%oo%', '%go%');
```

Comment:
nit: could you capitalize SQL keywords, e.g., SELECT, FROM ..., where possible?

Reply:
Done